packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1036,6 +1045,53 @@ export class Project extends ProjectBase {
     };
   }
 
+  private renderLogs(props: ProjectProps): CfnProject.LogsConfigProperty {
+    let s3Config: CfnProject.S3LogsConfigProperty|undefined;
+    let cloudwatchConfig: CfnProject.CloudWatchLogsConfigProperty|undefined;
+
+    if (props.logging?.s3) {
+      const s3Logs = props.logging.s3;
+
+      let status = 'ENABLED';
+      if ('enabled' in s3Logs) {
+        status = s3Logs.enabled ? 'ENABLED' : 'DISABLED';
+      }
+
+      s3Config = {
+        status,
+        location: `${s3Logs.bucket.bucketName}${s3Logs.prefix}`,
+        encryptionDisabled: s3Logs.encrypted ?? false,
```

Comment:
What's the default here in CloudFormation? We should probably not set this always. I assume the default is `false`, so just leave this as `encryptionDisabled: s3Logs.encrypted`, so that it's not filled in the template.

Reply:
default is false yes.